utils/oomparser/oomparser.go

```diff
@@ -153,12 +153,37 @@ func (self *OomParser) analyzeLines(ioreader *bufio.Reader, outStream chan *OomI
 				line = <-lineChannel
 			}
 			in_oom_kernel_log = false
+			glog.V(1).Infof("Sending an oomInstance: %v", oomCurrentInstance)
 			outStream <- oomCurrentInstance
 		}
 	}
 	glog.Infof("exiting analyzeLines")
 }
 
+func callJournalctl() (io.ReadCloser, error) {
+	cmd := exec.Command("journalctl", "-f")
```

Comment:
I think we need to do the generic solution. We can just use the pkg I referred to earlier. I am fine with going with journalctl binary approach first as this would work with kubernetes at least.